CONTRIBUTING.md

```
@@ -110,6 +110,31 @@ Semantic versioning and releases are automated in CI (e.g., `python-semantic-rel
 - Ensure all checks pass (tests and pre-commit) before requesting review.
 - Reference related issues and task IDs where applicable.
 
+### PR Title Format
+
+PR titles are validated automatically and must follow this format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)